apps/aehttp/src/aehttp_api_router.erl

```diff
@@ -37,7 +38,9 @@ path(Path0) ->
 method(Atom) ->
     list_to_binary(string:uppercase(atom_to_list(Atom))).
 
-is_enabled(Target, Tags) when is_atom(Target) ->
-    is_enabled(atom_to_binary(Target, utf8), Tags);
-is_enabled(Target, Tags) ->
-    lists:member(Target, Tags).
+is_enabled(Target, Tags, EnabledGroups) when is_atom(Target) ->
+    is_enabled(atom_to_binary(Target, utf8), Tags, EnabledGroups);
+is_enabled(Target, Tags, EnabledGroups) ->
+    lists:member(Target, Tags) andalso
+    lists:any(fun(Tag) -> lists:member(Tag, EnabledGroups) end, Tags).
```

Comment:
So if multiple tags then if one enabled enables all? This shall be (confirmed and) documented in the description in the schema.

Reply:
This is on an endpoint level. If a certain endpoint has a couple of tags and if any of them is enabled - this specific endpoint is enabled.